src/amp/components/Analytics.tsx

```diff
@@ -58,7 +58,7 @@ export const Analytics: React.FC<{
                      "vars": {
                        "title": "${title}",
                        "platform": "AMP",
-                       "sectionId": "${section}",
+                       "sectionId": "${section || 'undefined'}",
```

Comment:
Do we want 'undefined' as a string? I would expect to either see `undefined` the value or '' as an empty string here

Reply:
+1, that'd be classic showing `undefined` under section name on all articles 😆